src/containers/Articles/Article/RatingContainer.jsx

```diff
@@ -0,0 +1,103 @@
+/* eslint-disable import/prefer-default-export */
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+
+import Rating from '../../../components/Rating';
+import { rateFetch, updateRateValue } from '../../../store/actions/async/rate';
+import { infoToast, successToast } from '../../../components/UI/Toast/toast';
+
+
+export class RatingContainer extends Component {
+    state = {
+        rate_score: 0
```

Comment:
> We should use `camelCase`

@collinewait I had to change this back to snake case because in the backend i am supplying rate_score to the endpoint in the post as data in snake case.

Reply:
I get the point @missvicki. We can use camelCase and then convert to snake style in an action(`updateRateValue`) before sending the data to the API. When we do that, there will be one instance of snake style and it will be in code that interacts with the API.